app/models/foreman_tasks/task.rb

```diff
@@ -85,8 +85,26 @@ def execution_type
       delayed? ? N_('Delayed') : N_('Immediate')
     end
 
+    def get_humanized(method)
```

Comment:
Needed to define this for tests to work properly: we've already seen this issue in Katello, so I though it's better to fix it right way.